kazu/ontology_preprocessing/autocuration.py

```diff
@@ -146,6 +146,31 @@ def initial_lowercase_then_upper_to_case_sensitive(
         return resource
 
 
+def multiword(
+    resource: OntologyStringResource,
+) -> OntologyStringResource:
+    """If any synonym has more than one words, it's likely a noun phase and we should
+    mark all synonyms PROBABLE.
+
+    :param resource:
+    :return:
+    """
+    if any(" " in syn.text for syn in resource.original_synonyms):
```

Comment:
Could this also trigger if a synonym has an unnecessary whitespace that wasn't stripped or is that not possible?

For example `diabetes ` would trigger it but that is not the idea of this function, maybe we should use `.split` and count occurrences that way?

Reply:
it could... I'll add a strip to eliminate the possibility